analyzers/tests/SonarAnalyzer.UnitTest/Rules/AllBranchesShouldNotHaveSameImplementationTest.cs

```diff
@@ -18,12 +18,10 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-extern alias csharp;
-extern alias vbnet;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using SonarAnalyzer.UnitTest.TestFramework;
-using CSharp = csharp::SonarAnalyzer.Rules.CSharp;
-using VisualBasic = vbnet::SonarAnalyzer.Rules.VisualBasic;
+using CSharp = SonarAnalyzer.Rules.CSharp;
+using VisualBasic = SonarAnalyzer.Rules.VisualBasic;
```

Comment:
Also everywhere. Should be doable with search&replace.
```suggestion
using CS = SonarAnalyzer.Rules.CSharp;
using VB = SonarAnalyzer.Rules.VisualBasic;
```